CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# v2.1.0
+
+Reaction v2.1.0 adds minor features and performance enhancements, fixes bugs and contains no breaking changes on top of v2.0.0.
```

Comment:
What do you think about "contains no breaking changes since v2.0.0"

Not sure it's even necessary to specify, but it's probably nice to include since we've had a habit of including breaking changes in many of our 2.0.0 release candidates